client/web/src/utilities/input.ts

```diff
@@ -45,6 +52,15 @@ export async function handleKeyUp(e: KeyboardEvent) {
 	}
 }
 
+export async function handleMouseDown(e: MouseEvent) {
+	if (dialogIsVisible() && e.target && !(e.target as HTMLElement).closest(".dialog-modal .floating-menu-content")) {
```

Comment:
The idea of `e.target && !(e.target as HTMLElement).closest(".dialog-modal .floating-menu-content")` takes a bit too much thinking. It would be nice to make it a named variable above, like `let clickedContent = e.target && ...`